src/utils/exponential-backoff.js

```diff
@@ -48,9 +48,9 @@ async function exponentialBackoff (callFunction, payload = {}, { errorMessage =
       const nextDelayTime = delayTime * DELAY_MULTIPLIER
 
       if (errorMessage) {
-        logger.error(errorMessage, { payload, delayTime, attemptsLeft, error })
+        logger.error(errorMessage, { payload, delayTime, attemptsLeft, error: error.message })
```

Comment:
Depending on the error, we may never receive the correct text. Adding `error.message` provides more info in the logs for exponential backoff.